core-bundle/config/listener.yaml

```diff
@@ -566,6 +566,13 @@ services:
         tags:
             - kernel.event_subscriber
 
+    contao.listener.transport_security_header:
+        class: Contao\CoreBundle\EventListener\TransportSecurityHeaderListener
+        arguments:
+            - '@contao.routing.scope_matcher'
+        tags:
+            - kernel.event_listener
```

Comment:
`autoconfigure` is enabled for our event listeners, so if you prefer you could omit this and instead use the `#[AsEventListener]` PHP attribute.

Reply:
We don't use that in any listener yet - I guess we should keep it unified.